pkgs/by-name/cl/clac/package.nix

```diff
@@ -2,32 +2,39 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  nix-update-script,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "clac";
-  version = "0.3.3-unstable-2021-09-06";
+  version = "0.3.4";
 
   src = fetchFromGitHub {
     owner = "soveran";
     repo = "clac";
-    rev = "beae8c4bc89912f4cd66bb875585fa471692cd54";
-    sha256 = "XaULDkFF9OZW7Hbh60wbGgvCJ6L+3gZNGQ9uQv3G0zU=";
+    tag = finalAttrs.version;
+    hash = "sha256-DcW35jKIZQqkNa5Y6am2e5/TAEg3Fo2n+fHG3nOpNzM=";
   };
 
   makeFlags = [ "PREFIX=$(out)" ];
 
   postInstall = ''
-    mkdir -p "$out/share/doc/clac"
-    cp README* LICENSE "$out/share/doc/clac"
+    mkdir -p $out/share/doc/clac
+    cp README* LICENSE $out/share/doc/clac
```

Comment:
Please keep the quotes.

Reply:
I didn't see quotes in other packages and we can be sure that it doesn't contain special characters which could mess things up, but I don't mind adding them back.